components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -267,11 +267,6 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     getSupportedWorkspaceClasses(): Promise<SupportedWorkspaceClass[]>;
     maySetTimeout(): Promise<boolean>;
     updateWorkspaceTimeoutSetting(setting: Partial<WorkspaceTimeoutSetting>): Promise<void>;
-
-    /**
-     * getIDToken - doesn't actually do anything, just used to authenticat/authorise
```

Comment:
First part is true, it doesn't do anything. Sadly, it doesn't even authenticate/authorise

Reply:
Removed it from the go side as well, as this didn't authorize anything at all